transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -361,17 +370,74 @@ public void init(IEditorSite site,
             if (!latestVersion) prefs.setValue(VERSION_PREFERENCE, version);
 
             // Ensure Maven will compile transformations folder
-            final IProject project = manager.project();
+            IProject project = manager.project();
 			File pomFile = project.getLocation().append("pom.xml").toFile(); //$NON-NLS-1$
-			final MavenUtils mavenUtils = new MavenUtils();
+			MavenUtils mavenUtils = new MavenUtils();
 			mavenUtils.addResourceFolder(project, pomFile, Util.TRANSFORMATIONS_FOLDER);
 			mavenUtils.addResourceFolder(project, pomFile, MavenUtils.RESOURCES_PATH);
 
+			// Ensure import of package com.sun.el (FUSETOOLS-2039)
+	        IPath pomPath = project.getRawLocation() != null
+	                        ? project.getRawLocation()
+	                        : ResourcesPlugin.getWorkspace().getRoot().getLocation().append(project.getFullPath());
+	        Model pomModel = MavenPlugin.getMaven().readModel(new File(pomPath.append("pom.xml").toOSString()));  //$NON-NLS-1$
+	        Build build = pomModel.getBuild();
+	        Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+	        Plugin plugin = pluginsByName.get("org.apache.felix:maven-bundle-plugin"); //$NON-NLS-1$
+	        if (plugin == null) {
+	            plugin = new Plugin();
+	            plugin.setGroupId("org.apache.felix"); //$NON-NLS-1$
```

Comment:
Not sure what's going on with the indentation.  It looks fine locally, and I've checked my prefs, and they're set to insert spaces for tabs.